src/index.html

```diff
@@ -3,12 +3,49 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
```

Comment:
Remember about proper formatting
```suggestion
    <meta 
    	name="viewport"
```